package.json

```diff
@@ -74,22 +88,29 @@
     "events": "^3.3.0",
     "github-markdown-css": "^5.1.0",
     "http-server": "^14.1.1",
-    "jest": "^29.3.1",
-    "jest-environment-jsdom": "^29.3.1",
-    "lerna": "^6.4.0",
+    "jest": "^29.4.1",
+    "jest-environment-jsdom": "^29.4.1",
     "lodash.merge": "^4.6.2",
     "markdown-it": "^13.0.1",
-    "prettier": "^2.5.1",
+    "nx": "18.0.4",
+    "prettier": "^2.6.2",
     "rimraf": "^3.0.2",
+    "rollup": "^4.9.5",
+    "rollup-plugin-auto-external": "^2.0.0",
+    "rollup-plugin-scss": "^4.0.0",
+    "rollup-plugin-svg": "^2.0.0",
+    "rollup-plugin-typescript2": "^0.36.0",
```

Comment:
I was testing multiple typescript plugins for rollup when I finally landed on this one, which worked.

I can test the other ones again, if you see the need.

Reply:
I'd prefer the "official" plugin, if you can make it work? Should this prove to be an issue/consume a lot of time, we can also roll with this underground one, but its declining popularity may bite later.

🎩 in case it "just doesn't work atm".

Reply:
I now remember why `@rollup/plugin-typescript` can not be used.  
If I read the documentation correctly, there is no option to not check with the rollup plugin for type errors. This leads to the problem, that it also want's to convert the vue files to js, which utterly fails.

It might be possible to change this behaviour once we upgrade to vue@3, as we'd just be using vite in that case, but for now I suggest that we keep using the unofficial plugin.